pkg/agent/packetcapture/packetcapture_controller.go

```diff
@@ -449,7 +449,14 @@ func (c *Controller) performCapture(
 	if err != nil {
 		return false, err
 	}
-	pcapngWriter, err := pcapgo.NewNgWriter(file, layers.LinkTypeEthernet)
+
+	// set SnapLength here to make tcpdump on Mac OSX works. By default, its value is
+	// 0 and means unlimited, but tcpdump on Mac OSX will complain:
+	// 'tcpdump: pcap_loop: invalid packet capture length <len>, bigger than snaplen of 524288'
+	defaultNgInterface := pcapgo.DefaultNgInterface
```

Comment:
the "default" prefix in your variable name doesn't make sense, given that you are actually modifying the library's defaults